### ISSUE TITLE
Move bindings into universalInjectables

### ISSUE THREAD

**Issue by litstrong** [litstrong/Dragon#66]
```
@Component({
  selector: 'login-selector',
  viewBindings: [HTTP_BINDINGS],
})
```

So here could remove "viewBindings" and put it into a universal bindings, then you don't need to bind everytime.

**Comment by litstrong** [comment]
```
var universalInjectables = [
  routerInjectables,
  HTTP_BINDINGS,
];
bootstrap(AppComponent, [universalInjectables]);
```

**Comment by litstrong** [comment]
Made mistakes in the comment of commit:
https://github.com/litstrong/Dragon/commit/423f3899b9683491de9a9764fe8183a44fe16711

**Comment by litstrong** [comment]
Should fix #66 not #57.
